core-rust/state-manager/src/store/rocks_db.rs

```diff
@@ -206,6 +207,16 @@ impl VersionedCf<Epoch, LedgerProof> for EpochLedgerProofsCf {
     type VersionedValue = VersionedLedgerProof;
 }
 
+/// Ledger proofs of protocol updates.
+/// Schema: `Epoch.to_bytes()` -> `scrypto_encode(VersionedLedgerProof)`
+/// Note: This duplicates a small subset of [`StateVersionToLedgerProof`]'s values.
```

Comment:
```suggestion
/// Note: This duplicates a small subset of [`LedgerProofsCf`]'s values.
```

_(plz also propagate the fix to my line that you copied :p)_